dotcom-rendering/.storybook/preview-head.html

```diff
@@ -0,0 +1,36 @@
+<script>
+	document.cookie = 'gu-cmp-disabled=true';
+</script>
+
+<!-- Add Curl Loader for interactive elements-->
+<script>
+	window.curlConfig = {
+		apiName: 'require',
+	};
+	window.curl = window.curlConfig;
+</script>
+<script defer="" src="/js/curl-with-js-and-domReady.js"></script>
+
+<style>
+	body {
+		margin: 0;
+		font-family: 'Guardian Text Egyptian Web';
+	}
+	figure {
+		margin: 1em 0;
+	}
+	figure.element-embed {
+		margin: 2em 0;
+	}
+	.js-email-sub__iframe {
+		width: 100%;
+	}
+	.js-email-sub__iframe + figcaption {
+		margin-top: -8px;
+	}
+	video,
+	.element-atom iframe,
+	.element-audio iframe {
+		width: 100%;
+	}
+</style>
```

Comment:
This seems very arbitrary… but I’m assuming is copied as-is?

Reply:
yeah i haven't audited, its just what was there previously